ground/gcs/src/plugins/pathplanner/flightdatamodel.cpp

```diff
@@ -517,10 +512,12 @@ void FlightDataModel::readFromFile(QString fileName)
         homelocField = homelocField.nextSibling();
     }
 
-    // For now, reset home location to the location in the file. Later we can see if we would
-    // rather ask the user whether to remap the flight plan to the current home location or use
-    // the one in the map
-    setHomeLocation(HomeLLA);
+    // For now, reset home location to the location in the file.
```

Comment:
They are not: we do reset the home location. The comment mentions that in a future release, we should let the user select whether to update the home location, or to remap the flight plan to the currently stored home location on the uav